src/account/validators.ts

```diff
@@ -0,0 +1,18 @@
+import Joi from "joi";
+import type { UpdateTagsRequest } from "./models";
+
+export const validateUpdateTagsRequest = (
+  data: unknown
+): data is UpdateTagsRequest => {
+  const validation = Joi.object()
+    .keys({
+      emailFromAuthToken: Joi.string().email().required(),
+      addTags: Joi.array().items(Joi.string()),
+      removeTags: Joi.array().items(Joi.string()),
+    })
```

Comment:
Should we validate that we either add or remove tags on each request? I guess it doesn't really matter, but it would be weird to call this endpoint with no tags at all.